aim/web/ui/src/modules/BaseExplorer/types.ts

```diff
@@ -66,8 +66,7 @@ export interface IUIComponents {
 export interface IQueryFormProps extends IBaseComponentProps {
   hasAdvancedMode?: boolean;
 }
-export interface IGroupingProps
-  extends Omit<IBaseComponentProps, 'visualizationName'> {}
+export interface IGroupingProps extends IBaseComponentProps {}
```

Comment:
The older code omits the visualizationName property from IBaseComponentProps, Im pretty sure the type will have that property by extending from IBaseComponentProps, why you added additional property, it should come with inheritance?

Reply:
Since in most cases we omit the `visualizationName` property, I removed it and add there where we really need it.